applications/nrf5340_audio/src/bluetooth/le_audio_cis_headset.c

```diff
@@ -445,7 +445,7 @@ static void stream_recv_cb(struct bt_bap_stream *stream, const struct bt_iso_rec
 		bad_frame = true;
 	}
 
-	uint32_t octets_per_frame = bt_codec_cfg_get_octets_per_frame(stream->codec);
+	uint32_t octets_per_frame = stream->qos->sdu;
```

Comment:
I think we should do similar to what we do in BIS, store the configuration at the start and then check against that all the way. That way we are also guarded against the stream suddenly changing without us knowing it. One more thing is that sdu might be more than one frame, so octets_per_frame might not be the correct name.